react/components/Map.js

```diff
@@ -335,35 +653,6 @@ class Map extends Component {
     return a[0] !== b[0] || a[1] !== b[1]
   }
 
-  handleMarkerPositionChange = newPosition => {
-    if (!this.geocoder) {
-      this.geocoder = new this.props.googleMaps.Geocoder()
-    }
-
-    this.geocoder.geocode(
-      { location: newPosition },
-      this.handleNewMarkerPosition
-    )
-  }
-
-  handleNewMarkerPosition = (results, status) => {
-    const { googleMaps, onChangeAddress, rules } = this.props
-
-    if (status === googleMaps.GeocoderStatus.OK) {
-      if (results[0]) {
-        const googleAddress = results[0]
-        const autoCompletedAddress = geolocationAutoCompleteAddress(
-          this.address,
-          googleAddress,
-          rules
-        )
-        onChangeAddress(autoCompletedAddress)
-      }
-    } else {
-      console.warn(`Google Maps Error: ${status}`)
-    }
-  }
-
   render() {
```

Comment:
`render` line 656... yup, one hell of massive component.

Reply:
🤷🏻‍♂️